apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -145,30 +147,38 @@ class Demosaic : public Halide::Generator<Demosaic> {
     }
 
     void schedule() {
-        int vec = get_target().natural_vector_size(UInt(16));
-        bool use_hexagon = get_target().features_any_of({Target::HVX_64, Target::HVX_128});
-        if (get_target().has_feature(Target::HVX_64)) {
-            vec = 32;
-        } else if (get_target().has_feature(Target::HVX_128)) {
-            vec = 64;
-        }
-        for (Func f : intermediates) {
-            f.compute_at(intermed_compute_at)
-                .store_at(intermed_store_at)
-                .vectorize(x, 2*vec, TailStrategy::RoundUp)
-                .fold_storage(y, 2);
-        }
-        output.compute_at(output_compute_at)
-            .vectorize(x)
-            .unroll(y)
-            .reorder(c, x, y)
-            .unroll(c);
-        if (use_hexagon) {
-            output.hexagon();
+#if 0
```

Comment:
Is this a dead code?

Reply:
Is there a way to send in the auto_schedule flag (from CameraPipe) when using the apply interface? I couldn't find any, and it's the reason I took the scheduling part out.

Reply:
@steven-johnson Do you have any suggestion on how to pass the auto_schedule flag with the apply<> interface?

Reply:
Hmm... as of right now, the apply<> interface allows for setting the Inputs, but doesn't provide a way to set GeneratorsParams (which is needed for auto_schedule). I think we'll have to add such a way. Let me think about it briefly.

Reply:
Please see https://github.com/halide/Halide/pull/2338 for the current best approach for apply<>

Reply:
Thanks; committed here now.